test/e2e/autoscale_test.go

```diff
@@ -129,8 +157,73 @@ func (suite *AutoscaleTestSuite) TestAutoScaleCollector() {
 	require.Greater(t, maxCollectorCount, 1, "Collector never scaled")
 }
 
+func getSimpleStreaming(name, namespace string) *v1.Jaeger {
+	kafkaClusterURL := fmt.Sprintf("my-cluster-kafka-brokers.%s:9092", kafkaNamespace) // FIXME this may need to change
```

Comment:
Why would this need to be changed?

Reply:
It doesn't, I thought I had taken out all TODOs and FIXME.  Originally I was thinking of creating a kafka instance programatically in the same namespace as the test, but using the kafka instance that we already create works fine and is less work.  So I will clean this up.